src/main.ts

```diff
@@ -23,7 +23,7 @@ const config: Types.Core.GameConfig = {
         default: "arcade",
         arcade: {
             gravity: { x: 0, y: 0 },
-            debug: false,
+            debug: true,
```

Comment:
```suggestion
            debug: window.location.search.includes('debugphysics')
```

then we can enable it any time by adding ?debugphysics to the url

Reply:
good idea. Applied